airflow/cli/commands/kubernetes_command.py

```diff
@@ -90,7 +90,7 @@ def cleanup_pods(args):
     print('Loading Kubernetes configuration')
     kube_client = get_kube_client()
     print(f'Listing pods in namespace {namespace}')
-    continue_token = None
+    continue_token = ''
     while True:  # pylint: disable=too-many-nested-blocks
         pod_list = kube_client.list_namespaced_pod(namespace=namespace, limit=500, _continue=continue_token)
         for pod in pod_list.items:
```

Comment:
If you want to go "Pythonic" route, you can do:

![image](https://user-images.githubusercontent.com/8811558/104321638-cedb9480-54db-11eb-82a8-a5b81dd0df29.png)

Reply:
Btw I have no problem with the current suggestion too of using an empty string (`''`) , but want to make sure it works with 11.0 and 12.0 (we will support 12.0 soon -- or at least in near future)

Personally, I would not set it for the first iteration but it does not make a difference

Reply:
@kaxil Thank you for your suggestion, I like this code, it is easier to test.